vars/testsPublishResults.groovy

```diff
@@ -1,4 +1,5 @@
 import static com.sap.piper.Prerequisites.checkScript
+import static com.sap.piper.BashUtils.quoteAndEscape as q
```

Comment:
I guess github warns us here about the spaces (didn't recognize them as such before clicking the suggestion button). I have never seen this in groovy. But as the test works, it must work.

Reply:
It's a regular feature: https://groovy-lang.org/structure.html#_static_import_aliasing
I think it is suitable for an application like this, because it makes the string templates more readable.